varia/sass/blocks/utilities/_editor.scss

```diff
@@ -25,6 +26,11 @@
 	color: #{map-deep-get($config-global, "color", "secondary", "default")};
 }
 
+.wp-block .has-tertiary-color,
+.has-tertiary-color {
+	color: #{map-deep-get($config-global, "color", "tertiary", "default")};
```

Comment:
@MaggieCabrera For this rule to get compiled tertiary color must be defined under `global-config` even though the theme doesn't have a tertiary option.

Child themes like Rivinton doesn't have a tertiary either in the palette. 
By using CSS variables, Varia theme can give it a css-variable and inherit it in child theme without worrying about a color and child themes may or may not define these CSS variables.
However, we need to introduce CSS variables in all the child themes.
```
"tertiary": (
  "default": var(--wp--preset--color--tertiary),
  "hover": var(--wp--preset--color--tertiary-hover),
),
```

If we have to maintain both, Varia must define tertiary and give it a value `none` and child themes need to set a value or none.